tests/test_representation_manager.hh

```diff
@@ -85,74 +84,94 @@ namespace rascal {
   };
 
   struct SortedCoulombTestData {
+    using Factory_t =
+        std::tuple<std::tuple<std::string>, std::tuple<double, bool, double>>;
+    using ManagerTypeHolder_t =
+        StructureManagerTypeHolder<StructureManagerCenters,
+                                   AdaptorNeighbourList, AdaptorStrict>;
     SortedCoulombTestData() {
       std::vector<std::uint8_t> ref_data_ubjson;
       internal::read_binary_file(this->ref_filename, ref_data_ubjson);
       ref_data = json::from_ubjson(ref_data_ubjson);
-      filenames = ref_data.at("filenames").get<std::vector<std::string>>();
-      cutoffs = ref_data.at("cutoffs").get<std::vector<double>>();
+      auto filenames = ref_data.at("filenames").get<std::vector<std::string>>();
+      auto cutoffs = ref_data.at("cutoffs").get<std::vector<double>>();
+
+      for (auto && filename : filenames) {
+        for (auto && cutoff : cutoffs) {
+          auto a0{std::make_tuple(filename)};
+          auto a1{std::make_tuple(cutoff, consider_ghost_neighbours, cutoff)};
+          this->factory_args.emplace_back(a0, a1);
+        }
+      }
     }
     ~SortedCoulombTestData() = default;
 
     // name of the file containing the reference data. it has been generated
     // with the following python code:
     /*
-    import ubjson
-    from copy import copy
-    import numpy as np
-    import sys, os
-    path = '/local/git/librascal/' # should be changed
+    import sys
+    path = os.path.abspath('/local/git/librascal') # should be changed
```

Comment:
Changed to what?
Maybe we can include a mechanism in the build where these paths are automatically replaced by the users path?
What do you think?

Reply:
moved these data generation thing into the script folder so it does not clog the tests